tests/demo_config_transclude/demo/copier.yml

```diff
@@ -0,0 +1 @@
+!include ../include_me.yml
```

Comment:
IMHO this test is invalid.

You have to test the worst case: what happens if you have to include 2 files that define the same keys?

Reply:
I am expanding the tests: #146